### PR TITLE
Clean up kubectl browse somewhat.

### PR DESCRIPTION
**Pull request by brendandburns**
Fixes https://github.com/Azure/azure-cli/issues/1477

### CODE REVIEW

File: src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_params.py [hunk 1]
```diff
@@ -49,6 +49,8 @@ def _get_default_install_location(exe_name):
 name_arg_type = CliArgumentType(options_list=('--name', '-n'), metavar='NAME')
 
 register_cli_argument('acs', 'name', arg_type=name_arg_type, help='ACS cluster name', completer=get_resource_name_completion_list('Microsoft.ContainerService/ContainerServices'))
+register_cli_argument('acs', 'resource_group', arg_type=CliArgumentType(options_list=('--resource-group', '-g'),), help='Name of the Azure resource group')
```
Comment (derekbekoe) [inline src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_params.py]: Use `resource_group_name_type` here to keep this argument consistent throughout the CLI.
There are examples of it being used in this file.

File: src/command_modules/azure-cli-acs/setup.cfg [hunk 2]
```diff
@@ -1,2 +0,0 @@
-[bdist_wheel]
```
Comment (derekbekoe) [inline src/command_modules/azure-cli-acs/setup.cfg]: Can you keep this file in?
It's used for building the package.

File: src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_params.py (hunk 1 above)
Comment (brendandburns) [inline src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_params.py]: done.

File: src/command_modules/azure-cli-acs/setup.cfg (hunk 2 above)
Comment (brendandburns) [inline src/command_modules/azure-cli-acs/setup.cfg]: Removed, sorry, not sure what happened here...

**Comment by brendandburns** [comment]
Comments addressed, please re-check.

Thanks!

File: src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_params.py [hunk 3]
```diff
@@ -49,6 +49,8 @@ def _get_default_install_location(exe_name):
 name_arg_type = CliArgumentType(options_list=('--name', '-n'), metavar='NAME')
 
 register_cli_argument('acs', 'name', arg_type=name_arg_type, help='ACS cluster name', completer=get_resource_name_completion_list('Microsoft.ContainerService/ContainerServices'))
+register_cli_argument('acs', 'resource_group', arg_type=resource_group_name_type, help='Name of the Azure resource group')
```
Comment (derekbekoe) [inline src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_params.py]: You shouldn't need to override the help here as it's defined by `resource_group_name_type` for consistency throughout the CLI.
By defining help here, this help for `--resource-group` will have a different description than everywhere else.

Comment (brendandburns) [inline src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_params.py]: done.

**Pull request merged**